tasks/job/run.go

```diff
@@ -207,6 +207,15 @@ func (t *Task) runContainer(ctx *context.ExecuteContext) error {
 
 func (t *Task) createOptions(ctx *context.ExecuteContext, name string) docker.CreateContainerOptions {
 	interactive := t.config.Interactive
+	var dockerdevices []docker.Device
+	for _, dev := range t.config.Devices {
+		dockerdevices = append(dockerdevices,
+			docker.Device{
+				PathInContainer:   dev.Container,
+				PathOnHost:        dev.Host,
+				CgroupPermissions: dev.Permissions,
+			})
+	}
```

Comment:
Minor style thing: could you make this a separate function like `asPortBindings()`, so that `createOptions()` stays relatively small